microsoft-edge/extensions/publish/create-dev-account.md

```diff
@@ -33,109 +61,190 @@ To add and manage users in the Microsoft Edge program to manage extensions, you
 
 
 <!-- ---------- -->
 ###### Company account
 
-A company account is associated with an organization or business.  The account verification process is longer and involves confirmation that you're authorized to create the account for your company.  The duration of the process can range from a few days to a few weeks.  Your company might receive phone calls from Microsoft verification partners.
+A company account is associated with an organization or business.  The account verification process is longer, and involves confirmation that you're authorized to create the account for your company.  The duration of the process can range from a few days to a few weeks.  Your company might receive phone calls from Microsoft verification partners.
 
 For a company account, it's crucial to verify your Microsoft Edge program information when you enroll into a new Partner Center program.  This verification is needed to publish an extension at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com).  See [Verify your company account information](./verify-microsoft-edge-program.md).
 
 
 <!-- ------------------------------ -->
-#### Publisher display name
+#### Publisher display name (Company name)
+
+In the **Publisher display name (Company name)** field, enter the company name to display at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com):
 
-This field contains the name that's displayed at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com).  To use a particular name, that name must be available, and you must have the rights to use it.  Company accounts must use the registered business name of your organization.
+![Company name](./create-dev-account-images/company-name.png)
+
+To use a particular name, that name must be available, and you must have the rights to use it.  Company accounts must use the registered business name of your organization.
 
 The maximum length for this field is fifty (50) characters.
 
 
 <!-- ------------------------------ -->
-#### Contact details
+#### Contact info
 
-This field contains any contact information that Microsoft uses to contact you about any account issues.  After registration is complete, you receive an email confirmation.  For a company account, you must use the registered email address associated with your organization.
+The **Contact info** section contains any contact information that Microsoft uses to contact you about any account issues:
 
+![Contact info](./create-dev-account-images/contact-info.png)
 
-<!-- ------------------------------ -->
-#### Company approver
+For a company account (rather than an individual account), you must provide the contact information of your company approver, in all of the fields in this section.
+
+The **Contact info** section contains the following fields:
+* **First name** - For a company account, this must be the company approver name that's associated with your organization.
```

Comment:
is this true of all of the fields, or only these 4 fields?
rendered:
https://review.learn.microsoft.com/microsoft-edge/extensions/publish/create-dev-account?branch=pr-en-us-3667#contact-info

draft doc:
<img width="607" height="123" alt="image" src="https://github.com/user-attachments/assets/0bce2581-c64f-40e0-94fb-1cd1dd8cd3bb" />

This article doesn't have a **Company approver** heading/section.
AFAIK, the form at Partner Center doesn't have such a section.
This part of this article is mostly expressed in terms of: headings = section labels of the form.